src/Audio/Listener.cs

```diff
@@ -69,28 +69,28 @@ public static Vector3f UpVector
         }
         
         #region Imports
-        [DllImport("csfml-audio-2", CallingConvention = CallingConvention.Cdecl), SuppressUnmanagedCodeSecurity]
```

Comment:
and sadly, when converting the project to .Net Core you will not have the same compatibility
see [this](https://docs.microsoft.com/en-us/dotnet/standard/net-standard#net-platforms-support) so user will need to at least target .NET Framework 4.5